docs/02-getting-started/02-installation.md

```diff
@@ -131,3 +131,19 @@ There are many more editors out there and some have varying degrees of rust supp
 unless you have a strong preference to another editor (e.g. Sublime, Emacs, Vim) trying one of the two above is
 recommended, especially if you are new to rust. Once you are confident in the language, you can always use another
 editor and customize it to your liking.
+
+[CosmWasm IDE](https://github.com/oraichain/cw-vscode)
+
+CosmWasm IDE is a tool that simplifies the CosmWasm smart contract development & deployment processes. It integrates with Gitpod & Keplr to create a simple yet powerful environment to build, deploy, and interact with the CosmWasm smart contracts through default and custom networks using CosmWasm. The tool is currently maintained by Oraichain & CosmWasm.
```

Comment:
Can you also mention `with gitpod you can develop on the browser and it is well maintained.`